convert_to_pdbfile.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+# author          : Alina Kutlushina
+# date            : 15.05.2020
+# license         : BSD-3
+#==============================================================================
+
+__author__ = 'alina'
+
+import os
+import argparse
+import numpy as np
+from rdkit import Chem
+from rdkit.Chem import AllChem
+from multiprocessing import Pool
+from read_input import read_input
+
+
+def read_molfiles(input, output_path):
+    for mol, mol_name in read_input(input):
+        yield mol, mol_name, output_path
+
+
+def map_write_pdb(args):
+    return write_pdb_file(*args)
+
+
+def write_pdb_file(mol, mol_name, output_path):
+    mol = Chem.AddHs(mol)
+    if '2D' in Chem.MolToMolBlock(mol):
```

Comment:
test for 2D is wrong, replace with this code
`if mol.GetNumConformers() == 0 or not mol.GetConformer(0).Is3D()`